examples/games/game_menu.rs

```diff
@@ -27,6 +52,14 @@ enum DisplayQuality {
 #[derive(Resource, Debug, Component, PartialEq, Eq, Clone, Copy)]
 struct Volume(u32);
 
+fn default_font(font_size: f32, color: Color) -> TextStyle {
```

Comment:
Nit: Move this function below `fn main` so it can be with the other helper functions.

Reply:
Moved